mm/Kconfig

```diff
@@ -191,6 +191,49 @@ config MM_KASAN
 		bugs in native code. After turn on this option, Please
 		add -fsanitize=kernel-address to CFLAGS/CXXFLAGS too.
 
+config MM_UBSAN
+	bool "Undefined Behavior Sanitizer"
+	default n
+	depends on ARCH_ARM || ARCH_RISCV
+	---help---
+		UBSan is a fast undefined behavior detector. UBSan modifies
+		the program at compile-time to catch various kinds of
+		undefined behavior during program execution
+
+if MM_UBSAN
+
+config MM_UBSAN_SHIFT
```

Comment:
remove, enable by -fsanitize=undefined from:
https://bcain-llvm.readthedocs.io/projects/clang/en/latest/UndefinedBehaviorSanitizer/

Reply:
Enable full UBSan may lead to a huge image that is hard to run on real device, so I think it's better to let the user determine the check item.

Reply:
If so, we need list all possible USBSan option which is too boring. Let's add MM_UBSAN_ALL like PR #6885.

Reply:
OK